pkg/secrets-store/secrets-store.go

```diff
@@ -94,12 +88,12 @@ func newIdentityServer(d *csicommon.CSIDriver) *identityServer {
 }
 
 // Run starts the CSI plugin
-func (s *SecretsStore) Run(ctx context.Context, driverName, nodeID, endpoint, providerVolumePath, minProviderVersions, grpcSupportedProviders string, client client.Client) {
+func (s *SecretsStore) Run(ctx context.Context, driverName, nodeID, endpoint, providerVolumePath, minProviderVersions string, providerClients map[string]*CSIProviderClient, client client.Client) {
 	klog.Infof("Driver: %v ", driverName)
 	klog.Infof("Version: %s", vendorVersion)
 	klog.Infof("Provider Volume Path: %s", providerVolumePath)
 	klog.Infof("Minimum provider versions: %s", minProviderVersions)
-	klog.Infof("GRPC supported providers: %s", grpcSupportedProviders)
+	// klog.Infof("GRPC supported providers: %s", providerClients)
```

Comment:
nit: Could we remove this line if not used? And can we log the `--grpc-supported-providers` flag in `main.go`. It'll be useful for debugging.